### DISCUSSION TITLE
Convert data from HDF5 to BMP or Netcdf

### DISCUSSION THREAD

**Discussion by oki36** [ARM-DOE/pyart#1302] [Q&A]
Hi everyone!
I am currrently doing a research studying storms. I need to use radar data for this. The data is in HDF5, so i have to convert it to BMP or netcdf, i am new in using python and this library "Py-art"  . Does anyone can help me with an example script?

**Comment by zssherman** [comment]
Hello! So depending on which format in HDF5 it is in, sinarme, odim etc, you'll use one of the aux io readers. Example:

```
radar = pyart.aux_io.read_odim_h5('filename')

# output to netCDF
pyart.io.write_cfradial('output_filename', radar, format='NETCDF4')
```

**Reply by oki36** [reply]
Thank yoy very much! The data is in HDF5 gamic. 
So, i can still write my script like that right?

Also, is the same if i use wradlib?

**Comment by kmuehlbauer** [comment]
@oki36 

Yes, but you would have to use the Py-ART gamic reader:

```python
radar = pyart.aux_io.read_gamic('filename')
```

I'd say it is not the same as using wradlib, as you get back numpy arrays and metadata or xarray Datasets from wradlib. There might be a bit more involved to convert to netcdf as with Py-ART radar-object where you have "batteries" included.

Just a heads up, we ([openradar](https://github.com/openradar)) have a new discourse discussion platform (https://openradar.discourse.group/) for any kind of questions related to weather radar. 

You might also want to follow the developments of https://github.com/openradar/xradar, a new radar reader/writer based on xarray/xarray-datatree.

**Reply by oki36** [reply]
Thank you very much! I'll try using it.
and also I'll follow those platforms.. Thank you.
